atom/browser/login_handler.h

```diff
@@ -40,7 +40,7 @@ class LoginHandler : public base::RefCountedThreadSafe<LoginHandler> {
   // thread.
   content::WebContents* GetWebContents() const;
 
-  const net::AuthChallengeInfo* auth_info() const { return &auth_info_; }
+  const net::AuthChallengeInfo* auth_info() const { return &*auth_info_; }
```

Comment:
This works (and I used it myself when prototyping ideas) but `auth_info_.get()` is probably better for production due to being more straightforward/clearer

Reply:
Oh yeah!

Reply:
Just saw this comment https://github.com/electron/electron/pull/15002#discussion_r223451769

Reply:
I'd be OK with returning a scoped refptr too :+1: